packages/commonwealth/client/scripts/controllers/app/webWallets/polkadot_web_wallet.ts

```diff
@@ -52,26 +55,40 @@ class PolkadotWebWalletController
     return injector.signer;
   }
 
+  public async getChainId() {
+    return Promise.resolve(app.chain?.id || this.defaultNetwork);
+  }
+
+  public async getRecentBlock(chainIdentifier: string) {
+        // TODO: are we using the polkadot API anywhere else on the frontend?
```

Comment:
Yes, everywhere. See the substrate controller. However, be careful here -- this is a very heavyweight operation.

Reply:
We've decided not to sign blocks for Substrate so this will be removed, we can just return null here without making the RPC call

Reply:
@jnaviask discussed with Bob today, will sync up with you offline about sourcing blocks for signing